src/pages/Cart/CartPage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { CartCard } from './CartCard/CartCard';
+import './CartPage.scss';
+import { useHistory } from 'react-router-dom';
+import { MyContext } from '../../App';
+import { GoBack } from '../../components/GoBack/GoBack';
+
+export const CartPage = () => {
+  const { cart, setCart } = useContext(MyContext);
+  const stringedCart = cart.map(item => JSON.stringify(item));
```

Comment:
first of all, cart is a singular name and you store an array there...
secondly, you make too many iterations until you finally get cards array sorted

Reply:
I'm not just sorting an array, I get an array without duplicate objects.